src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -449,8 +501,13 @@ public void setTalonMode(NeutralMode mode) {
    * @since 1/30/2023
    */
   private void updateOdometry() {
+    Pose2d currentPose = odometry.getPoseMeters();
     odometry.update(navx.getRotation2d(), getLeftDistance(), getRightDistance());
-    field.setRobotPose(odometry.getPoseMeters());
+    field.setRobotPose(currentPose);
+
+    SmartDashboard.putNumber("Current X", currentPose.getX());
+    SmartDashboard.putNumber("Current Y", currentPose.getY());
+    SmartDashboard.putNumber("Current Angle", currentPose.getRotation().getDegrees());
```

Comment:
Same with these, there is already a field, are these needed?

Reply:
ditto